login/service.go

```diff
@@ -840,6 +843,30 @@ func (keycloak *KeycloakOAuthProvider) equalsKeycloakUserProfileAttributes(ctx c
 	return profileEqual, nil
 }
 
+func fillUserFromResponse(userProfile oauth.UserProfile, identity *account.Identity) (bool, error) {
```

Comment:
looks like the `bool` return value is never used

Reply:
Yeah we previously used it for checking if keycloak needs to be updated, we are getting rid of it.